datahub-web-react/src/app/entity/glossaryTerm/profile/GlossaryRelatedTermsResult.tsx

```diff
@@ -56,12 +62,19 @@ export default function GlossaryRelatedTermsResult({ glossaryRelatedTermType, gl
                         <Typography.Title style={{ margin: '0' }} level={3}>
                             {glossaryRelatedTermType}
                         </Typography.Title>
-                        <Button type="text" onClick={() => setIsShowingAddModal(true)}>
-                            <PlusOutlined /> Add Terms
-                        </Button>
+                        {relatedTermsMutable && (
+                            <Button type="text" onClick={() => setIsShowingAddModal(true)}>
+                                <PlusOutlined /> Add Terms
+                            </Button>
+                        )}
                     </TitleContainer>
                     {glossaryRelatedTermUrns.map((urn) => (
-                        <RelatedTerm key={urn} urn={urn} relationshipType={relationshipType} />
+                        <RelatedTerm
+                            key={urn}
+                            urn={urn}
+                            relationshipType={relationshipType}
+                            mutable={relatedTermsMutable}
```

Comment:
and similarly we could change this prop from `mutable` to `isEditable` (just cuz we usually use the verb "edit" instead of "mutate" for this type of thing and prefer descriptive booleans like `isEditable` instead of just `editable`)